src/main/java/org/openmrs/addonindex/domain/artifactory/ArtifactoryFolderInfoChild.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.addonindex.domain.artifactory;
+
+import org.openmrs.addonindex.util.Version;
+
+/**
+ * Represents a single version(uri) and folder status( whether the entry is a folder or not)
```

Comment:
In retrospect, putting javadoc here doesn't really help. Just say "@see ArtifactoryFolderInfo"